lib/suspenders/generators/production/manifest_generator.rb

```diff
@@ -10,6 +10,7 @@ def render_manifest
           scripts: {},
           env: {
             APPLICATION_HOST: { required: true },
+            AUTO_MIGRATE_DB: { value: "true" },
```

Comment:
Note: this will make PR apps auto-migrate. I'm not sure if this should be here or not, we don't use it at CommonLit and prefer for a migration-carrying PR to have engineers migrate if/when necessary.